src/platforms/flutter/troubleshooting.mdx

```diff
@@ -10,10 +10,12 @@ If you need help solving issues with Sentry's Flutter SDK, you can read the edge
 
 ## Known Limitations
 
-- Flutter `split-debug-info` and `obfuscate` flags aren't yet supported on iOS; they are supported only on Android. If this feature is enabled, Dart stack traces are not human readable, this is a tooling limitation, See: [43612](https://github.com/dart-lang/sdk/issues/43612) and [43274](https://github.com/dart-lang/sdk/issues/43274).
+- Flutter `split-debug-info` and `obfuscate` flags aren't yet supported on iOS/macOS; they are supported only on Android. If this feature is enabled, Dart stack traces are not human readable, this is a tooling limitation, See: [43612](https://github.com/dart-lang/sdk/issues/43612) and [43274](https://github.com/dart-lang/sdk/issues/43274).
 - If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.
 - Also, Issue's titles might be obfuscated as we rely on the `runtimeType`, but they may not be human-readable, See: [Obfuscate Caveat](https://flutter.dev/docs/deployment/obfuscate#caveat)
 - Layout related errors are only caught by [FlutterError.onError](https://api.flutter.dev/flutter/foundation/FlutterError/onError.html) in debug mode. In release mode, they are removed by the Flutter framework. See [Flutter build modes](https://flutter.dev/docs/testing/build-modes).
+- Windows and Linux do not have a native integration. This means that there is no device information included in crash reports.
```

Comment:
I don't think we need this, it'd be the same for Web, which is covered by `Limited support for Flutter Web, Windows and Linux`